package/SKSE/Plugins/CommunityShaders.json

```diff
@@ -1,88 +1,94 @@
 {
```

Comment:
Trying to figure out why the diff is so bad. Did you lint the json? Maybe we need a action to also prettify it.